sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -629,8 +628,8 @@
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>3.1.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
         <configuration>
-          <configLocation>eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml</configLocation>
-          <suppressionsLocation>eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml
+          <configLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml</configLocation>
+          <suppressionsLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml
```

Comment:
These configLocation and suppressionsLocation entries should be exactly the same, the pathing between the two modified entries is different. Also, by that note there a number of other items that may need to be updated for the pathing
- headerLocation entries [1](https://github.com/Azure/azure-sdk-for-java/blob/f691e525e75585111b3f5e8af693e49e7ff36fb1/sdk/parents/azure-client-sdk-parent/pom.xml#L143) and [2](https://github.com/Azure/azure-sdk-for-java/blob/f691e525e75585111b3f5e8af693e49e7ff36fb1/sdk/parents/azure-client-sdk-parent/pom.xml#L634)
- spotbugs [excludeFilterFile](https://github.com/Azure/azure-sdk-for-java/blob/f691e525e75585111b3f5e8af693e49e7ff36fb1/sdk/parents/azure-client-sdk-parent/pom.xml#L668)
There are also other things that may need to get modified like the [non-shipping-modules](non-shipping-modules). If so then pom.data.xml would also need to get updated for its non-shipping modules.

Reply:
Yep. I've got local changes I haven't pushed yet, I'm stepping through all this. It's kinda messy.